ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -3863,6 +3863,127 @@ module VBD = struct
           )
           (fun () -> cleanup_attached_vdis vm (id_of vbd))
 
+  let unplug task vm vbd force =
```

Comment:
I confirmed that this is the same as prior to the commit referenced:
`git diff 1a46f33^ -- ocaml/xenopsd/xc/xenops_server_xen.ml`, it doesn't show any change in `let unplug` (it of course shows let deactivate/etc.).

Of course now the risk is that we bugfix detach/deactivate or unplug and we lose that fix when we drop the other, but I think this is just a temporary situation to fix a regression until we find a better solution.